### ISSUE TITLE
undisplay parts with no data

### ISSUE THREAD

**Issue by dthauvin** [metagriffin/modseclogc#1]
hi Metagriffin 
Thanks for this tool . 

Is it possible to add a parameter that hide the "PARTS" when it's empty ? For example the command below display a lot of empty line because i have no "rule" modsec triggrered in my log file .
# modseclogc --show-parts K modsec.logsecu

.......
--d86e6b12-K--

--95c52a3c-K--

--e7eb404f-K--

--cd51e70c-K--

......

Thanks

**Comment by metagriffin** [comment]
yes, i think this does make sense, but i'd probably make it an explicit option (since that deviates from standard modsec logging behavior). can you provide a sample input file and expected output file that i can use as a unit test?

**Comment by dthauvin** [comment]
Hi 
Here is a file that I cleaned.

Thanks
[modsec_audit.log.txt](https://github.com/metagriffin/modseclogc/files/284604/modsec_audit.log.txt)
